src/lib/ng-packagr.ts

```diff
@@ -27,23 +27,25 @@ export async function createNgPackage(opts: NgPackagrCliArguments): Promise<void
   let buildDirectoryRoot: string;
   try {
     // READ `ng-package.json` and obtain model, as well as secondary packages
-    const ngPackages: PackageSearchResult = await discoverPackages(opts.project);
+    const ngPackages = await discoverPackages(opts.project);
 
-    const rootPackage: NgPackageData = ngPackages.rootPackage;
+    const rootPackage = ngPackages.rootPackage;
```

Comment:
I am not sure if these are intentional or not aka, your coding style.
But typescript doesn't need these.